### PR TITLE
Transfer Event

### PR DESCRIPTION
**Pull request by vivienfan**
This PR adds yml for transfer of owership & custody event (a subevent extended from `Event`).

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

File: docs/openapi/components/schemas/common/TransferEvent.yml [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+  portOfDestination:
+    title: Port of Destination
+    description: 
+    $ref: ./Place.yml
+    $linkedData:
+      term: place
+      '@id': https://schema.org/Place
+  countryOfDestination:
+    title: Country of Destination
+    description: Country in which the product(s) were transfered to.
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/TransferEvent.yml:98]:
```suggestion
    description: Country to which the product(s) were transferred.
```

File: docs/openapi/components/schemas/common/TransferEvent.yml [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+          - TransferCustody
+  eventId:
+    type: string
+  eventTime:
+    type: string
+  place:
+    title: Place
+    description: Information on where the event took place.
+    $ref: ./Place.yml
+    $ref: ./Place.yml
```
Comment (nissimsan) [inline docs/openapi/components/schemas/common/TransferEvent.yml:29]: Once should be enough.  :)

**Review by nissimsan — APPROVED** [review]
I'm intrigued by this idea, @vivienfan! Have you given any thoughts as to how this will work with traditional bill of lading title transfers? Tons of potential - I'd love to see/discuss a workflow which illustrates this process.

**Review by mprorock — APPROVED** [review]

**Review by mkhraisha — APPROVED** [review]

**Comment by OR13** [comment]
CI is failing due to JSON parse issue in the example

**Comment by brownoxford** [comment]
@vivienfan can you please fix so that CI is passing?

**Pull request merged**